Release/src/websockets/client/ws_client_wspp.cpp

```diff
@@ -333,7 +333,8 @@ class wspp_callback_client : public websocket_client_callback_impl,
                     websocket_incoming_message incoming_msg;
 
                     incoming_msg.m_msg_type = websocket_message_type::ping;
-                    incoming_msg.m_body = concurrency::streams::container_buffer<std::string>(msg);
+                    // 'move' the payload into a container buffer to avoid any copies.
+                    incoming_msg.m_body = concurrency::streams::container_buffer<std::string>(std::move(msg));
```

Comment:
`std::move` is redundant here. Compiler will ignore it because msg is const ref. To use `std::move` the msg should be &&:
`std::string&& msg`